site/en/blog/page-lifecycle-api/index.md

```diff
@@ -862,93 +811,76 @@ detect the `pagehide` event and only use `unload` if the browser doesn't support
 
 The difference between `beforeunload` and `unload`, though, is that there are
 legitimate uses of `beforeunload`. For instance, when you want to warn the user
 that they have unsaved changes they'll lose if they continue unloading the page.
 
 Since there are valid reasons to use `beforeunload` but using it prevents pages
-from being added to the Back-Forward Cache, it's recommended that you _only_
+from being added to the back/forward cache, it's recommended that you _only_
 add `beforeunload` listeners when a user has unsaved changes and then remove
 them immediately after the unsaved changes are saved.
 
 In other words, don't do this (since it adds a `beforeunload` listener
 unconditionally):
 
 ```js
-addEventListener(
-  'beforeunload',
-  event => {
-    // A function that returns `true` if the page has unsaved changes.
-    if (pageHasUnsavedChanges()) {
-      event.preventDefault();
-      return (event.returnValue = 'Are you sure you want to exit?');
-    }
-  },
-  {capture: true}
-);
+addEventListener('beforeunload', (event) => {
+  // A function that returns `true` if the page has unsaved changes.
+  if (pageHasUnsavedChanges()) {
+    event.preventDefault();
+    return (event.returnValue = 'Are you sure you want to exit?');
+  }
+});
 ```
 
 Instead do this (since it only adds the `beforeunload` listener when it's
 needed, and removes it when it's not):
 
 ```js
-const beforeUnloadListener = event => {
+const beforeUnloadListener = (event) => {
   event.preventDefault();
   return (event.returnValue = 'Are you sure you want to exit?');
 };
 
 // A function that invokes a callback when the page has unsaved changes.
 onPageHasUnsavedChanges(() => {
-  addEventListener('beforeunload', beforeUnloadListener, {capture: true});
+  addEventListener('beforeunload', beforeUnloadListener);
 });
 
 // A function that invokes a callback when the page's unsaved changes are resolved.
 onAllChangesSaved(() => {
-  removeEventListener('beforeunload', beforeUnloadListener, {capture: true});
+  removeEventListener('beforeunload', beforeUnloadListener);
 });
 ```
 
-{% Aside %}
-  [PageLifecycle.js](https://github.com/GoogleChromeLabs/page-lifecycle) library provides the convenience methods `addUnsavedChanges()`
-  and `removeUnsavedChanges()`, that follow all the best practices
-  outlined above. They're based on a [draft proposal](https://docs.google.com/document/d/1SXY2zGDgh7L73kX3bXfxCi15IV_Z7RnH9lRIZnuqf6g/edit?usp=sharing) to officially replace the `beforeunload` event with a declarative API that can be less easily
-  abused and more reliable on mobile platforms.
-
-  If you want to use the `beforeunload` event properly and in a
-  way that works cross-browser, the [PageLifecycle.js](https://github.com/GoogleChromeLabs/page-lifecycle)
-  library is our recommended solution.
-{% endAside %}
-
 ## FAQs
 
-### My page does important work when it's hidden, how can I stop it from being frozen or discarded?
+**Why isn't there a "loading" state?**
```

Comment:
Stylistic nit and you're free to reject it: but it seems there should be some introductory text in this section, and the questions should probably be another heading level deep (as they're already functioning visually as a header anyway).

Reply:
I believe these are consistent throughout. The use of `<em>` is to visually separate these inline notes from the other  content.